packages/playground/src/dashboard/components/user_nodes.vue

```diff
@@ -110,6 +110,14 @@
         />
         <SetExtraFee class="me-2" :nodeId="item.raw.nodeId" />
       </template>
+
+      <template v-slot:[`item.country`]="{ item }">
+        {{ item.raw.country || "-" }}
+      </template>
+
+      <template v-slot:[`item.serialNumber`]="{ item }">
+        {{ item.raw.serialNumber || "-" }}
+      </template>
```

Comment:
Do we really need this? we already set the value to "-" in the same file below.

Reply:
the below one related to card details itself not the table